Makefile

```diff
@@ -217,6 +217,18 @@ test-modules-%:
 		&& go run . -p $(subst _,/,$(subst test-modules-,,$@)) ${EACHMODULE_FLAGS} \
 		"go test -timeout=1m ${UNIT_TEST_TAGS} ./..."
 
+cachedep: cachedep-modules-.
+
+cachedep-modules-%:
```

Comment:
When should we use this task? Should this be a part of generate task?

Reply:
think this is used by automation

Reply:
Yes this will be used by automation that I am working on for caching dependency artifacts.